### PR TITLE
Fix click-to-show

### PR DESCRIPTION
**Pull request by codemonkey800**
## Description

https://airtable.com/shr6MXADGlNNaohN7/tblKHsTqQzLZW70S9/viwhDGIqzDkq0EprV/recrlK9jQC4NC4iYj

Fixes the click-to-show button for the new theme. This works by injecting Jupyter Book stylesheets into the application and fixing the styles to work with our theme.

## Demo

~https://napari-org-fix-click-to-show.vercel.app/tutorials/index.html~

https://user-images.githubusercontent.com/2176050/134994422-b56db573-6b9e-4759-a96e-a30f8978b3d2.mp4

### CODE REVIEW

**Comment by codemonkey800** [comment]
@liaprins-czi I tagged you in case you had any feedback on the click-to-show button's UI 😄

**Review by liaprins-czi — APPROVED** [review]
Looks grand to me!

File: theme/napari/page.html [hunk 1]
```diff
@@ -18,6 +18,27 @@
...
+        {% endfor -%}
+      {%- endblock regular_styles -%}
+
+      {#- Theme-related stylesheets -#}
+      {%- block theme_styles %}
+      {%- endblock -%}
+
+      {%- block extra_styles %}
+      {%- endblock -%}
+    {%- endblock styles -%}
```
Comment (codemonkey800) [inline theme/napari/page.html]: This is copied from the [Furo theme](https://github.com/pradyunsg/furo/blob/5cdb35c041dabc6dc651cbd4b2f84226f2d24528/src/furo/theme/furo/base.html#L51-L72) 😆

Comment (kne42) [reply theme/napari/page.html]: if possible, could you put this directly in as a comment?

File: theme/src/components/App/App.module.scss [hunk 2]
```diff
@@ -59,18 +59,69 @@
...
+        @apply m-0 w-6 h-6;
+      }
     }
 
+    // Animate copy button into view by changing opacity to 100%.
     .highlight:hover .copybtn {
       @apply opacity-100;
     }
+
+    // Page content sections.
```
Comment (kne42) [inline theme/src/components/App/App.module.scss:72]: could you add a clarifying comment to specify that these are overriding jupyterbook defaults and why we need to do so?

**Review by kne42 — APPROVED** [review]
LGTM! a few small comments :)

**Pull request merged**